[2024-04-04T09:22:56.543+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: real_estates.transform_parse_address manual__2024-04-04T08:47:03.043176+00:00 [queued]>
[2024-04-04T09:22:56.562+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: real_estates.transform_parse_address manual__2024-04-04T08:47:03.043176+00:00 [queued]>
[2024-04-04T09:22:56.564+0000] {taskinstance.py:2193} INFO - Starting attempt 3 of 3
[2024-04-04T09:22:56.592+0000] {taskinstance.py:2217} INFO - Executing <Task(_PythonDecoratedOperator): transform_parse_address> on 2024-04-04 08:47:03.043176+00:00
[2024-04-04T09:22:56.610+0000] {standard_task_runner.py:60} INFO - Started process 4552 to run task
[2024-04-04T09:22:56.618+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'real_estates', 'transform_parse_address', 'manual__2024-04-04T08:47:03.043176+00:00', '--job-id', '226', '--raw', '--subdir', 'DAGS_FOLDER/real_estates/__init__.py', '--cfg-path', '/tmp/tmpvqr4mp43']
[2024-04-04T09:22:56.627+0000] {standard_task_runner.py:88} INFO - Job 226: Subtask transform_parse_address
[2024-04-04T09:22:57.043+0000] {task_command.py:423} INFO - Running <TaskInstance: real_estates.transform_parse_address manual__2024-04-04T08:47:03.043176+00:00 [running]> on host 05a350db99a0
[2024-04-04T09:22:57.560+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='real_estates' AIRFLOW_CTX_TASK_ID='transform_parse_address' AIRFLOW_CTX_EXECUTION_DATE='2024-04-04T08:47:03.043176+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-04T08:47:03.043176+00:00'
[2024-04-04T09:22:57.570+0000] {__init__.py:98} WARNING - Town 千住宮元町 not in database
[2024-04-04T09:22:57.572+0000] {__init__.py:98} WARNING - Town 元橋本町 not in database
[2024-04-04T09:22:57.574+0000] {__init__.py:98} WARNING - Town 深大寺東町 not in database
[2024-04-04T09:22:57.578+0000] {__init__.py:98} WARNING - Town 莇生町 not in database
[2024-04-04T09:22:57.580+0000] {__init__.py:98} WARNING - Town 元総社町 not in database
[2024-04-04T09:22:57.581+0000] {__init__.py:98} WARNING - Town 中泉町 not in database
[2024-04-04T09:22:57.583+0000] {__init__.py:98} WARNING - Town 藤原光町 not in database
[2024-04-04T09:22:57.585+0000] {__init__.py:98} WARNING - Town ひうみ町 not in database
[2024-04-04T09:22:57.587+0000] {__init__.py:98} WARNING - Town 新井町 not in database
[2024-04-04T09:22:57.589+0000] {__init__.py:98} WARNING - Town 南町 not in database
[2024-04-04T09:22:57.590+0000] {__init__.py:98} WARNING - Town 散田町 not in database
[2024-04-04T09:22:57.593+0000] {__init__.py:98} WARNING - Town 並木西町 not in database
[2024-04-04T09:22:57.595+0000] {__init__.py:98} WARNING - Town 須惠町 not in database
[2024-04-04T09:22:57.597+0000] {__init__.py:98} WARNING - Town 若葉町 not in database
[2024-04-04T09:22:57.599+0000] {__init__.py:98} WARNING - Town 本木東町 not in database
[2024-04-04T09:22:57.601+0000] {__init__.py:98} WARNING - Town 出川町 not in database
[2024-04-04T09:22:57.604+0000] {__init__.py:98} WARNING - Town 菖蒲町 not in database
[2024-04-04T09:22:57.606+0000] {__init__.py:98} WARNING - Town 中町 not in database
[2024-04-04T09:22:57.607+0000] {__init__.py:98} WARNING - Town 新町 not in database
[2024-04-04T09:22:57.609+0000] {__init__.py:98} WARNING - Town 追分町 not in database
[2024-04-04T09:22:57.610+0000] {__init__.py:98} WARNING - Town 緑町 not in database
[2024-04-04T09:22:57.612+0000] {__init__.py:98} WARNING - Town 宮本町 not in database
[2024-04-04T09:22:57.614+0000] {__init__.py:98} WARNING - Town 北町 not in database
[2024-04-04T09:22:57.616+0000] {__init__.py:98} WARNING - Town 西戸部町 not in database
[2024-04-04T09:22:57.618+0000] {python.py:204} INFO - Done. Returned value not shown
[2024-04-04T09:22:57.748+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=real_estates, task_id=transform_parse_address, execution_date=20240404T084703, start_date=20240404T092256, end_date=20240404T092257
[2024-04-04T09:22:57.833+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-04-04T09:22:58.186+0000] {taskinstance.py:3312} INFO - 2 downstream tasks scheduled from follow-on schedule check
